src/app/components/normal-component/pick-game/pick-game.component.ts

```diff
@@ -58,6 +59,8 @@ class GameDescription {
 
     public static readonly LINES_OF_ACTION: Localized = () => $localize`Regroup your pieces to win. But your possible moves will often change!`;
 
+    public static readonly MARTIAN_CHESS: Localized = () => $localize`Win points by capturing pieces, but piece ownership is position related!`;
```

Comment:
but you only control pieces on your side of the board? Ça fait très abstrait/complexe là

Reply:
Ah bon ? Oui je veux bien, mais c'est aussi sa carachtéristique la plus originale donc bon :D
 
 Tu as une idée ? "Like the chess would behave on mars!"

Reply:
oui oui, mais ma proposition garde ce sens ?
Sinon en quoi les échecs seraient différents sur mars ? Ça n'a pas d'sens !

Reply:
La proposition garde quel sens ??

Reply:
proposition = ce que j'ai proposé (= "but you only control pieces on your side of the board")
sens = ce que tu as dit dans le message précédant le mien (= "c'est aussi sa carachtéristique la plus originale")
clair luc, ne pas ?

Reply:
Ah, y avais eu un malentendu, ta "but you only control pieces on your side of the board" était tellement plus clair que la mienne qu'en lisant le premier message je me suis dit "qu'est ce qu'il veut que je dise d'autres" sans me rapeller que j'avais écrit "ownership is position related" xD KRRRR.
Gedaan meneer